go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/couler-proj/couler
 
-go 1.15
+go 1.13
```

Comment:
at least 1.14?

Reply:
1.13 is inline with internal version as well as Argo Workflows. Is there anything interesting/useful in particular in 1.14 that’s worth bringing?

Reply:
nvm